Formula/ivtools.rb

```diff
@@ -0,0 +1,22 @@
+class Ivtools < Formula
+  desc "X11 vector graphic servers"
+  homepage "http://www.ivtools.org"
+  url "https://github.com/vectaport/ivtools/archive/refs/tags/ivtools-2.0.11.tar.gz"
+  sha256 "d94e39937aa4d86671bbd9fbc3bf38b53439ead724b3761c90155c820ca9349a"
+  license "MIT"
+
+  depends_on "ace" => "7.0.5"
```

Comment:
```suggestion
  depends_on "ace"
```
The `=> "7.0.5"` doesn't do anything.

Reply:
Thanks.  I have just made a new ivtools release with the change to do the homebrew build against the installed versions of the ACE lib and include directories.